motor/providers/container/container.go

```diff
@@ -110,14 +113,28 @@ func NewDockerEngineImage(endpoint *providers.Config) (ContainerProvider, error)
 		return nil, err
 	}
 
+	labelImageIdFull := strings.Split(ii.Labels["docker.io/digests"], ",")[0]
+	labelImageId := strings.Split(labelImageIdFull, "@")[1]
+
 	log.Debug().Msg("found docker engine image " + ii.ID)
+	// This is the image id that is used to pull the image from the registry.
+	log.Debug().Msg("found docker engine image " + labelImageId)
+	if ii.Size > 1024 { // > 1GB
+		log.Warn().Int64("size", ii.Size).Msg("image is bigger than 1GB, this will take some time")
```

Comment:
This is only to demonstrate, that we could act upon the image size. We could create a bufferedReader for images which are smaller than X. We would than need to talk about how large X is.